src/cartogram_info/read_csv.cpp

```diff
@@ -31,13 +19,13 @@ int extract_id_header_col_index(
 
 int extract_area_col_index(
   const csv::CSVReader &reader,
-  const argparse::ArgumentParser &arguments)
+  const std::optional<std::string> area_col_name)
 {
   // Find index of column with target areas. If no area column header was
   // passed with the command-line flag --area, the area column is assumed to
   // have index 1
   int area_col = 1;
-  if (auto area_header = arguments.present<std::string>("-A")) {
+  if (auto area_header = area_col_name) {
```

Comment:
We should keep styling consistent here with optionals. Should be changed to `is_area_header`. Or we change everything to something else altogether. I'll make this change in the commit I make to this pull request.